modules/postgres/16.2.bcr.1/source.json

```
@@ -0,0 +1,9 @@
+{
+    "url": "https://ftp.postgresql.org/pub/source/v16.2/postgresql-16.2.tar.gz",
+    "integrity": "sha256-K4IBBH7IGs0brSnbonjXiOeJG5w+gjLtoWuyneyBMcc=",
+    "strip_prefix": "postgresql-16.2",
+    "overlay": {
+        "BUILD.bazel": "sha256-8QE0ahABYpRl3YJ1z5ys5no/xrSLwRqQx5n5ONGlYPc=",
+        "MODULE.bazel": "sha256-V9h0YZCp7DY6C3NQx7DAILdS4FJ7jWIN838tX24aamU="
```

```suggestion
        "MODULE.bazel": "sha256-hv47e8n0LmKET80UBaRl6rayE0t+e1BWBZfzVOCANx0="
```

This would fix the "Checked in MODULE.bazel file doesn't match the one in the extracted and patched sources" error. Because the overlay you specified has a wrong checksum, it didn't end up actually overlaid, thus causing the error.